### ISSUE TITLE
Bug on Affidavit for Care or Custody

### ISSUE THREAD

**Issue by maevemacglinchey** [SuffolkLITLab/docassemble-MAVirtualCourt#552]
I was running through the [Affidavit of Care or Custody](https://apps.suffolklitlab.org/interview?i=docassemble.MAAffidavitDisclosingCareOrCustody%3Adata%2Fquestions%2Fstandalone.yml#page1) that's live on the site and ran into [this error](https://apps.suffolklitlab.org/interview?i=docassemble.MAAffidavitDisclosingCareOrCustody%3Adata%2Fquestions%2Fstandalone.yml#page14) when I selected a court and then selected 'next':

```
Error
Retry

AttributeError: 'NoneType' object has no attribute 'name'
Log
Traceback (most recent call last):
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 6540, in assemble
    exec_with_trap(question, user_dict)
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 7813, in exec_with_trap
    exec(the_question.compute, the_dict)
  File "<code block>", line 15, in <module>
NameError: name 'fill_in_court_name' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/webapp/server.py", line 6979, in index
    interview.assemble(user_dict, interview_status, old_user_dict, force_question=special_question)
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 6820, in assemble
    raise the_error
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 6611, in assemble
    question_result = self.askfor(missingVariable, user_dict, old_user_dict, interview_status, seeking=interview_status.seeking, follow_mc=follow_mc, seeking_question=seeking_question)
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 7251, in askfor
    exec_with_trap(question, user_dict)
  File "/usr/share/docassemble/local3.6/lib/python3.6/site-packages/docassemble/base/parse.py", line 7813, in exec_with_trap
    exec(the_question.compute, the_dict)
  File "<code block>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'name'
```

Cc: @nonprofittechy

**Comment by knod** [comment]
Thanks for finding this! Which court did you select? Did you select it from the drop-down?

**Comment by maevemacglinchey** [comment]
I have recorded myself replicating this issue twice:
- [video with no prior cases](https://drive.google.com/file/d/1cAsTHSai2btN93v2nGSIlQhVOZFaOLuM/view?usp=sharing)
- [video with a prior divorce case](https://drive.google.com/file/d/1LOyGG0A32UuQe5lqiXKyYs83w0_9Oq_8/view?usp=sharing)

In both recordings, I tried to select a different court but got an error every time.

Cc: @nonprofittechy @CaroRob @knod @colarusso

**Comment by nonprofittechy** [comment]
Fixed in 0.8
